tests/components/zha/zha_devices_list.py

```diff
@@ -784,7 +784,7 @@
             },
             ("binary_sensor", "00:11:22:33:44:55:66:77-1-6"): {
                 "channels": ["on_off"],
-                "entity_class": "Opening",
+                "entity_class": "Motion",
```

Comment:
The entity class is called OnOffMotion, you need to update it, otherwise the test would fail

Reply:
I already fixed that locally, but now I'm having issues on test_registries.py:265:
```
FAILED tests/components/zha/test_registries.py::test_weighted_match[random manufacturer-mock model-OnOffModel] - AssertionError: assert 'OnOffMultimodel' == 'OnOffModel'
FAILED tests/components/zha/test_registries.py::test_weighted_match[mock manufacturer-mock model-OnOffModelManufacturer] - AssertionError: assert 'OnOffMultimodel' == 'OnOffModelManufacturer'

Results (545.71s):
     628 passed
       2 failed
         - tests/components/zha/test_registries.py:265 test_weighted_match[random manufacturer-mock model-OnOffModel]
         - tests/components/zha/test_registries.py:265 test_weighted_match[mock manufacturer-mock model-OnOffModelManufacturer]
```

Reply:
Push it and I'll take a look later.

Reply:
Fixed now :)

Reply:
That was a subtle bug